src/app/libp2p_helper/src/libp2p_helper/app.go

```diff
@@ -8,35 +8,33 @@ import (
 	"math"
 	"os"
 	"strconv"
-	"sync"
 	"time"
 
 	ipc "libp2p_ipc"
 
 	capnp "capnproto.org/go/capnp/v3"
 	"github.com/go-errors/errors"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
 	net "github.com/libp2p/go-libp2p/core/network"
 	peer "github.com/libp2p/go-libp2p/core/peer"
-	pubsub "github.com/libp2p/go-libp2p-pubsub"
 	mdns "github.com/libp2p/go-libp2p/p2p/discovery/mdns"
 	"github.com/multiformats/go-multiaddr"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 func newApp() *app {
-	outChan := make(chan *capnp.Message, 1<<12) // 4kb
+	outChan := make(chan *capnp.Message, 1<<12) // 4096 messages stacked
 	ctx := context.Background()
 	return &app{
 		P2p:                      nil,
 		Ctx:                      ctx,
-		Subs:                     make(map[uint64]subscription),
-		Topics:                   make(map[string]*pubsub.Topic),
-		ValidatorMutex:           &sync.Mutex{},
-		Validators:               make(map[uint64]*validationStatus),
-		Streams:                  make(map[uint64]net.Stream),
+		_subs:                    make(map[uint64]subscription),
```

Comment:
Why the underscore prefixes? We don't do this in other parts of the code.

Reply:
This is @tizoc's decision, I guess to visually discourage usage of these fields.

We may roll back the prefixes, I agree they look a bit alien to the rest of the codebase.
Though I liked the idea of making use of these fields visually uncomfortable, so thought why not

Reply:
> I guess to visually discourage usage of these fields.

That was exactly it. I am not terribly familiar with Go, so I guess that naming convention is not idiomatic at all. From what I found, the proper way would be to separate this into a separate package to make those fields truly private.

Reply:
:+1: on moving to a different package later